hosted_control_planes/hcp-disconnected/disconnected-install-ibmz-hcp.adoc

```diff
@@ -0,0 +1,31 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="disconnected-install-ibmz_{context}"]
+include::_attributes/common-attributes.adoc[]
+= Deploying {hcp} on {ibm-z-title} in a disconnected environment
+:context: disconnected-install-ibmz-hcp
+
+toc::[]
+
+{hcp-capital} deployments in disconnected environments function differently than in a standalone {product-title}.
+
+{hcp-capital} involves two distinct environments:
+
+* Control plane: Located in the management cluster, where the {hcp} pods are run and managed by the Control Plane Operator.
+* Data plane: Located in the workers of the hosted cluster, where the workload and a few other pods run, managed by the Hosted Cluster Config Operator.
+
+The `ImageContentSourcePolicy` (ICSP) for the data plane is managed through the `ImageContentSources` API in the hosted cluster manifest. 
+
+For the control plane, ICSP objects are managed in the management cluster*. These objects are parsed by the HyperShift Operator and are shared as `registry-overrides` entries with the Control Plane Operator. These entries are injected into any one of the available  deployments in the {hcp} namespace as an argument.
+
+To work with disconnected registries in the {hcp}, you must first create the appropriate ICSP in the management cluster. Then, to deploy disconnected workloads in the data plane, you need to add the entries that you want into the `ImageContentSources` field in the hosted cluster manifest.
+
+.Prerequisites to deploy {hcp} on {ibm-z-title} in a disconnected environment
+
+* You set up the mirror registry. For more information, see "Creating a mirror registry with mirror registry for Red Hat OpenShift".   
+* You mirrored an image for a disconnected installation. For more information, see "Mirroring images for a disconnected installation using the oc-mirror plugin".
+
```

Comment:
Add an "Additional resources" section here to provide links to the two topics that you mention in the prerequisites:
```
[role="_additional-resources"]
.Additional resources
* xref:../../disconnected/mirroring/installing-mirroring-creating-registry.adoc#mirror-registry-introduction_installing-mirroring-creating-registry[Creating a mirror registry with mirror registry for Red Hat OpenShift]

* xref:../../disconnected/mirroring/installing-mirroring-disconnected.adoc#installing-mirroring-disconnected[Mirroring images for a disconnected installation using the oc-mirror plugin]
```